hummingbot/market/binance/binance_market.pyx

```diff
@@ -298,47 +296,20 @@ cdef class BinanceMarket(MarketBase):
             del self._account_available_balances[asset_name]
             del self._account_balances[asset_name]
 
-    async def _update_trade_fees(self):
-        cdef:
-            double current_timestamp = self._current_timestamp
-
-        if current_timestamp - self._last_update_trade_fees_timestamp > 60.0 * 60.0 or len(self._trade_fees) < 1:
-            try:
-                res = await self.query_api(self._binance_client.get_trade_fee)
-                for fee in res["tradeFee"]:
-                    self._trade_fees[fee["symbol"]] = (Decimal(fee["maker"]), Decimal(fee["taker"]))
-                self._last_update_trade_fees_timestamp = current_timestamp
-            except asyncio.CancelledError:
-                raise
-            except Exception:
-                self.logger().network("Error fetching Binance trade fees.", exc_info=True,
-                                      app_warning_msg=f"Could not fetch Binance trading fees. "
-                                                      f"Check network connection.")
-                raise
-
-    cdef object c_get_fee(self,
-                          str base_currency,
-                          str quote_currency,
-                          object order_type,
-                          object order_side,
-                          object amount,
-                          object price):
-        cdef:
-            object maker_trade_fee = Decimal("0.001")
-            object taker_trade_fee = Decimal("0.001")
-            str trading_pair = base_currency + quote_currency
-
-        if order_type.is_limit_type() and fee_overrides_config_map["binance_maker_fee"].value is not None:
+    @staticmethod
+    def c_get_fee(base_currency: str,
```

Comment:
Is it possible to not change this function from cython to python and keep it static? ie.
```python
@staticmethod
cdef object c_get_fee(...
```

we use `c_` prefix only for cython functions.

Reply:
Cython static methods cannot be called by instance name, it wouldn't even compile.
Using `instance.__class__` to access cython static method compiles but produces error at runtime.

I suggest we stick with `c_estimate_fee` as a python static class.
Let me know if you know a way around accessing cython static method from instance names.

Reply:
<img width="960" alt="cython_static" src="https://user-images.githubusercontent.com/31972210/83706507-91428f00-a60f-11ea-89ac-a064877b1864.PNG">

That is what I'm talking about.
Doing `market.__class__.c_estimate_fee(is_maker)` would compile but give error at runtime.

Reply:
should I make it python static again?

Reply:
hmm, perhaps making the method static is not quite feasible.
we might get into a bit over engineering here.
how about, for this task, we simply create a python module called `estimate_fee.py` (this can be in `hummingbot/core/utils`), it has a function called 'estimate_fee' and what it does is returning the fee based on market name and is_maker parameter? 
then all the connectors just call this function to get its fee, the paper trade market also call this function to get the fee. we then don't have to make any changes to strategy_base and market_base classes, no changes to c_get_fee signature. We can then leave out the refactoring for later. 
What do you think?

Reply:
Very good, nice plan.
I'll work with that and delete this branch and make a new PR.

Reply:
For DEX, are we good to go with having a fixed flat fee/gas for each exchange?

Reply:
@vic-en Yes, just a flat fee for each exchange, amount and token name, e.g. 0.00001 ETH. we will need to have fee overrides configuration for this as well so users can update this.

Reply:
Alright. I'll make commits soon.